minicluster/src/main/java/alluxio/underfs/UnderFileSystemCluster.java

```diff
@@ -113,14 +113,12 @@ public static UnderFileSystemCluster getUnderFilesystemCluster(String baseDir,
   }
 
   /**
-   * This method is only used by the alluxio.client.FileOutStreamIntegrationTest unit-test.
+   * Gets the UnderFileSystemClass.
```

Comment:
for getters,  you can simply make it like 
```java
/** 
 * @return the {@link UnderFileSystem} class name
 */
```

Also in this codebase, we always trying to use `{@link}` when possible (to help potential IDE refactor)